CHANGELOG.md

```diff
@@ -14,8 +14,19 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 
 ## v2.0.0 (WIP)
 
-- Removed the `wharfapi.Provider.UploadURL` field, which will be removed from
-  the `Provider` struct in `github.com/iver-wharf/wharf-api` in v5.0.0. (#21)
+- BREAKING: Updated to work with wharf-api v5.0.0. Functions now
+  also match the names used in wharf-api. (#29)
+
+- BREAKING: Removed the `wharfapi.Provider.UploadURL` field, which was
+  removed from the `Provider` struct in wharf-api v5.0.0. (#21)
+
+- Changed version of `github.com/iver-wharf/wharf-core` from v1.2.0 to v1.3.0.
+  (#29)
+
+- Added dependencies: (#29)
+
+  - `github.com/google/go-querystring` v1.1.0
+  - `github.com/iver-wharf/wharf-api` v5.0.0
```

Comment:
If I get this straight, if we were to create a sort of version matrix, it would be like:

| wharf-api-client-go | wharf-api |
| :---: | :---: |
| v1 | v0 &rarr; v5 |
| v2 | v5 |

I think such a matrix table and a really clear warning in the changelog is needed.

It's not much of a breaking change to start supporting wharf-api v5. Rather, it's a breaking change that we not stop supporting wharf-api v4.

Suggest changing notes to something like:

```diff
-- BREAKING: Updated to work with wharf-api v5.0.0. Functions now
-  also match the names used in wharf-api. (#29)
+- BREAKING: Removed support for wharf-api v4 and below. (#29)

+- Added support for wharf-api v5. (#29)
```

Reply:
Good point 👍🏻

Is it `and v5` because we have the deprecated endpoints still in place?
Maybe it would be good to specify that it only partially supports v5, and does so fairly weirdly?

Unless we've tested and it works flawlessly, I believe there were issues like only having the option to PUT some things that should have been POSTable, breaking with `wharf-api/v5`. I'm a bit fuzzy on this though, it's been a while.

Something like

  | wharf-api-client-go | wharf-api |
  | :---: | :---: |
  | v1 | v0, v1, v2, v3, v4, *and* v5* |
  | v2 | v5, *and beyond* |

*: Partial support for `wharf-api/v5`. Most behavior differs from what `wharf-api-client-go/v1` documentation and comments suggest. **Strongly discouraged to use** `wharf-api-client-go/v1` **with** `wharf-api/v5`**!**

Perhaps?

Reply:
And would this then go under the `v2.0.0` changelogs, or above it with a header like `Version Compatibilities` or something so we can keep updating it with new releases?

Reply:
Added header `Compatibility with wharf-api` above version.